creates/uploadFile.js

```diff
@@ -11,7 +11,11 @@ const uploadFile = (z, bundle) => {
   // file will in fact be an url where the file data can be downloaded from
   // which we do via a stream created by NPM's request package
   // (form-data doesn't play nicely with z.request)
-  formData.append('file', request(bundle.inputData.file));
+  formData.append('file', request(bundle.inputData.file),
+    { 
+        filename: bundle.inputData.filename
```

Comment:
I think the actual issue/question may be why we'd need this since as #9 I think describes `form-data` should be able to get the filename from the `Content-Disposition` header send by our stashed files.

I wonder if this is a bug in `form-data` as:

```
$ curl -I https://zapier.com/engine/hydrate/1906050/.eJw9jUEOwiAQRe_y110MClY5RK9ACEwtQadNSzRN07sLG__y5b38A0m24iWwSxH20uurIVIdxsSv6MS_GbY6nzlVZYkjOoSJQ3aZd1j1oBsZqmyWwlJc2ZcWDFXLX78-N9jjn7cHUJvS5k4a5_kDbKUoRQ:1e5G8R:JkZ7bvmO5yGhd50ORtc5O13bdPU/
HTTP/1.1 200 OK
Content-Disposition: attachment; filename="Invoice 00000145804.pdf"
```

Has the correct filename.

Reply:
If `form-data` isn't able to read the filename from the header, then perhaps we should not just pass the file URL but also the filename, but that's quite a change.